Content.Shared/DeltaV/SpaceFerret/SpaceFerretComponent.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Content.Shared.Roles;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.DeltaV.SpaceFerret;
+
+[RegisterComponent]
+public sealed partial class SpaceFerretComponent : Component
+{
+    [DataField]
+    public string RoleIntroSfx = "";
+
+    [DataField]
```

Comment:
since the others are all required dont hardcode this and specify in yaml, then change to not legally distinct :trollface: